brianlong/www.validators.app
# 182271924 stakeblaze pool

#### What's this PR do?
- add new stake pool

#### How should this be manually tested?
- test on staging
- check if there are logos and links for the new pool
- check if gathered data makes sense
- check if apy is correct

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/182271924)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
